cmd/operator-sdk/run/local.go

```diff
@@ -143,6 +143,16 @@ func (c runLocalArgs) runAnsible() error {
 	if err := setupOperatorEnv(c.kubeconfig, c.watchNamespace); err != nil {
 		return err
 	}
+
+	// If ansible-roles-path flag is not null, set ANSIBLE_ROLES_PATH EnvVar with
+	if c.ansibleOperatorFlags != nil && len(c.ansibleOperatorFlags.AnsibleRolesPath) > 0 {
+		log.Info(fmt.Sprintf("set the value %v for environment variable %v.",
+			c.ansibleOperatorFlags.AnsibleRolesPath, aoflags.AnsibleRolesPathEnvVar))
+		if err := os.Setenv(aoflags.AnsibleRolesPathEnvVar, c.ansibleOperatorFlags.AnsibleRolesPath); err != nil {
+			return fmt.Errorf("failed to set %s environment variable: %v", aoflags.AnsibleRolesPathEnvVar, err)
+		}
+	}
+
```

Comment:
it was in the wrong place and was not been used. 
It is just valid for ansible and was been added in the Go run by mistake.